cmd/cmd-review.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"context"
+	"os"
+	"runtime"
+
+	"github.com/lindell/multi-gitter/internal/multigitter"
+	"github.com/spf13/cobra"
+)
+
+// ReviewCmd reviews pull requests
+func ReviewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "review",
+		Short:   "Review pull requests.",
+		Long:    "Review pull requests with a specified branch name in an organization and with specified conditions.",
+		Args:    cobra.NoArgs,
+		PreRunE: logFlagInit,
+		RunE:    review,
+	}
+
+	cmd.Flags().StringP("branch", "B", "multi-gitter-branch", "The name of the branch where changes are committed.")
+	cmd.Flags().StringP("comment", "c", "", "Leave a review comment.")
+	cmd.Flags().BoolP("all", "a", false, "Review all pull requests in one go instead each individually.")
+	cmd.Flags().StringP("batch", "", "", "Review all pull requests without confirmation (--all is not required). Batch accepts one of [approve, reject, comment].")
+	cmd.Flags().BoolP("no-pager", "", false, "Do not use a pager for reviewing pull request diffs.")
+	cmd.Flags().BoolP("include-approved", "", false, "Include pull requests already approved by you.")
+	configurePlatform(cmd)
+	configureRunPlatform(cmd, false)
+	configureLogging(cmd, "-")
+	configureConfig(cmd)
+
+	return cmd
+}
+
+func review(cmd *cobra.Command, _ []string) error {
+	flag := cmd.Flags()
+
+	branchName, _ := flag.GetString("branch")
+	comment, _ := flag.GetString("comment")
+	all, _ := flag.GetBool("all")
+	batch, _ := flag.GetString("batch")
```

Comment:
Whatever it is called, it should be converted to an "enum" here, and therefore verified to be correct, before calling out to the multigitter package. 

See conflict strategy as an example

Reply:
Agreed with the enum, definitely makes sense 👍🏻
 
With the flag name, from a user perspective and cli tooling I would pick batch as its more obvious that something is going on which does not require user interaction which is not the case if it's called state. It could be --batch and --state=x. But thats an additional flag for no reason.

Reply:
I think --batch is more common to understand as a user that no additional follow up questions are required and it can be used silently. Might also call it silent instead.
If batch is just ditched and its called state I think it's not exactly what it's intended for.